config/confighttp/confighttp.go

```diff
@@ -31,8 +31,11 @@ import (
 	"go.opentelemetry.io/collector/extension/auth"
 )
 
-const headerContentEncoding = "Content-Encoding"
-const defaultMaxRequestBodySize = 20 * 1024 * 1024 // 20MiB
+const (
```

Comment:
should this be a separate PR?

Reply:
sgtm 

fixed in https://github.com/open-telemetry/opentelemetry-collector/pull/10877/commits/26bd5939717bc483f02229319cdf1158863fd524